test/unit/sources/alternates/source_test_helper.rb

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
Accidental newline.

I've put a rubocop config in the repo some time ago and have been chipping away at the violations as I encounter them around code I modify. The plan is to have it run as part of CI sometime, but that is still miles away.

New code should adhere to the style guide.